lib/dill/widgets/cucumber_methods.rb

```diff
@@ -0,0 +1,34 @@
+module Dill
+  module CucumberMethods
+    begin
+      require 'cucumber/ast/table'
+
+      # Compares this widget with the given Cucumber +table+.
+      #
+      # === Example
+      #
+      #   Then(/^some step that takes in a cucumber table$/) do |table|
+      #     widget(:my_widget).diff table
+      #   end
+      def diff(table, wait_time = Capybara.default_max_wait_time)
+        table.diff!(to_table) || true
+      end
+
+      def easy_diff(table, wait_time = Capybara.default_max_wait_time)
+        table = ArrayValue.new(table.raw).downcase_all
+        table = new_cucumber_table(table)
+
+        table.diff!(to_table.downcase_all) || true
+      end
```

Comment:
Would it make sense to have `diff` always be the entry point? So you'd have something like

```ruby
diff table, case_insensitive: true
```

There's that issue about people passing in an integer as the wait time, but I think it could be solved easily by detecting the type of the second argument. It would also enable you to implement something like `Dill.case_insensitive_table_comparisons = true`, to let people perform case insensitive comparisons by default, if they want to.